libs/sdk-ui-pivot/src/CorePivotTable.tsx

```diff
@@ -352,6 +357,23 @@ export class CorePivotTableAgImpl extends React.Component<ICorePivotTableProps,
         );
     }
 
+    /**
+     * Checks DOM if height has been set on PivotTable parent wrapper. If height has not been set,
+     * PivotTable will not be rendered correctly. Therefore, we need to inform the user at least on the console
+     * on what is happening.
+     *
+     * @private
+     */
+    private alertParentWrapperMissingHeight(): void {
+        const parentWrapper = document.getElementById(this.pivotTableId)?.parentElement;
+        const parentHeight = parentWrapper?.style?.height || "0";
+        if (isEmpty(parentHeight) || parseInt(parentHeight) < 20) {
```

Comment:
This will trigger false positives for for example "19em", no?

Reply:
Also, thanks to the `|| "0"` the `parentHeight` will never be empty.

Reply:
Good catch, unfortunately we don't differentiate em, px, %.... Instead I replaced checking styling by checking the element height itself.

Reply:
That is better :)